Makefile

```diff
@@ -4,7 +4,7 @@
 BASE_DIR:=$(dir $(realpath $(firstword $(MAKEFILE_LIST))))
 DIST_DIR:=$(BASE_DIR)dist/libraries
 
-GLOBAL_CFLAGS:=-O3
+GLOBAL_CFLAGS:=-O3 -s ENVIRONMENT=web,webview -s DOUBLE_MODE=0
```

Comment:
DOUBLE_MODE has been removed from Emscripten, see [here](https://github.com/emscripten-core/emscripten/blob/aab919ddfe92e88c8f1d67159d41e2291af7a1dd/src/settings.js#L2034), @dmitrylyzo can you remove it here?